test/unit_tests.html

```diff
@@ -2405,7 +2405,66 @@ <h4>Test HTML</h4>
         assert.equal($("#test-cancel-with-ic-on-success").text(), "");
       });
   </script>
-
+	
+  <div id="test-ic-push-url-and-ic-use-transition">
```

Comment:
Before or soon after merging, we will need add this test to the other *_unit_tests.html files, either using test/gen_tests.rb or manually pasting these added lines at the bottom of each.

(It looks like there's already one test from an earlier change that is missing from those other files, "Server Sent Event trigger handles missing ic-sse-src element". Whoever runs gen_tests.rb will see that one appear as well as the new one.)

Reply:
I'll do this myself after merging this pull request.